docs/sphinx/users/micromanager/index.txt

```diff
@@ -0,0 +1,47 @@
+Micro-Manager
+=============
+
+`Micro-Manager <https://www.micro-manager.org/wiki/Micro-Manager>`_ is a
+software framework for implementing advanced and novel imaging procedures,
+extending functionality, customization and rapid development of specialized
+imaging applications.
+
+Micro-Manager offers the functionality for saving the acquired images in
+Tiff/OME-Tiff format. Based on the mode of saving and the configuration
+settings, the acquired image can be saved with or without a companion file
+(:file:`*metadata.txt`):
+
+.. figure:: /images/micromanager_saving.png
+    :width: 50%
+    :align: center
+    :alt: Saving options in Micro-Manager
+
+
++---------------+----------+--------------+--------------+--------------------+
+|Saving         |          |Companion     |              |                    | 
+|Options within |          |File          |Bio-Formats   |                    |
+|Micro-Manager  |Format    |(metadata.txt)|Reading       | Reader Used        |
++===============+==========+==============+==============+====================+ 
+| Save as       |          |              |              |                    |
+| separate      |          |              |              |                    | 
+| image files   | Tiff     | Yes          | Full Support | MicromanagerReader |
++---------------+----------+--------------+--------------+--------------------+ 
+|               |          |              | Pixel data   |                    |
+|               |          |              | plus minimal |                    |
+|               | ome-tiff | No           | metadata*    | OMETiffReader      |
+| Save as image +----------+--------------+--------------+--------------------+
+| stack file    |          |              |              |                    |
+|               | ome-tiff | Yes**        | Full Support | MicromanagerReader |
++---------------+----------+--------------+--------------+--------------------+ 
+
+
+``*`` Minimal metadata: minimum amount of metadata required to validate
+OME-XML
```

Comment:
Discussed with @hflynn, we use `minimal` in a very specific semantics in the OME-TIFF specification documentation (minimal specification) and we might not want to confuse terms.